ci/README.md

```diff
@@ -0,0 +1,21 @@
+## Building the RPM locally
+
+```shell script
+./local-run.sh rpm
+```
+
+The artifacts will be located in dist/ folder in top-level project directory.
+
+## Build & uploading the package to PPA from the local machine
+
+```shell script
+./local-run.sh deb-ppa-upload
+```
+
+## Dockerized test against given git & Python version locally
+
+```shell script
+cp .env-sample .env  # and optionally edit if needed to change git/python version
+./local-run.sh tox
+```
+Note: optionally edit `ci/tox/.env-sample` file if needed to change git/python version.
```

Comment:
Hmmm `.env.sample` is **not** supposed to be edited 🤔 it's `.env` that's picked up by `docker-compose build`!

Reply:
Also, pls note that for that exact reason, `.env.sample` is tracked under git, while `.env` is git-ignored